### PR TITLE
Add maintenance notice

### PR DESCRIPTION
**Pull request by FunctionDJ**

### CODE REVIEW

**Comment by the-j0k3r** [comment]
This is not the reason why project is not maintained. Please revise.

**Comment by Podesta** [comment]
Afaik discourse has always had dark themes, but the options to allow users to enable them were limited by the admins of the forum. Can you confirm that dark mode is now available on all forums, irrespective of configs on the admin side?

**Comment by FunctionDJ** [comment]
@the-j0k3r that's not what the notice is for. it says that there is no maintainer, and that discourse has a built-in dark theme (if enabled) that could be used when looking for one. that's the reason i found this repo in the first place for example.
@Podesta [this post](https://meta.discourse.org/t/automatic-dark-mode-color-scheme-switching/161593) (a bit older) sounds like it's still an admin setting.

**Comment by FunctionDJ** [comment]
i understand that the native discourse dark theme and this project may not be related enough for the notice to apply. you may close my PR in that case.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+### :warning: This project is currently not maintained. Discourse has implemented their own Dark Mode which can be tried out at https://try.discourse.org/.
```
Comment (the-j0k3r) [inline README.md:1]:
```suggestion
### :warning: NOTICE This project is currently not maintained, [looking for contributors](https://github.com/StylishThemes/Discourse-Dark/issues/42). 

> NOTE: Discourse has implemented native Dark Mode which can be tried out at https://try.discourse.org/.
```

**Comment by the-j0k3r** [comment]
@FunctionDJ updated message to reflect status thx for contrib.

**Comment by the-j0k3r** [comment]
@FunctionDJ also if you notice, we already had a mention about the native dark styles on the readme https://github.com/StylishThemes/Discourse-Dark#using-discourse-type-built-in-dark-themes so we dont need anything else regarding this.

**Pull request merged**